src/components/Main/AgentInput/index.js

```diff
@@ -18,17 +20,75 @@ class AgentInput extends Component {
       : this.setState({ relWords: value })
   }
 
-  handleSubmit = (e) => {
+  handleSendEvent(e) {
     e.preventDefault();
+      this.setState({
+      currentChatMessage: ''
+    });
+  }
+  renderChatLog() {
+    return this.state.chatLogs.map((chat) => {
+      return (
+        <li key={`chat_${chat.id}`}>
+          <span className='chat-message'>{ chat.content }</span>
+          <span className='chat-created-at'>{ chat.created_at }</span>
+        </li>
+      );
+    })}
+    handleChatInputKeyPress(event) {
+      if(event.key === 'Enter') this.handleSendEvent(event);
+    }
+
+  updateCurrentChatMessage({target}) {
+      this.setState({
+        currentChatMessage: target.value
+      });
+  }
+    createSocket() {
```

Comment:
saving for later?

Reply:
I relocated this function but didnt want to delete it until I was sure the other location worked. Forgot to come back and delete this